src/components/topic/TopicDirective.js

```diff
@@ -11,12 +11,11 @@ goog.require('ga_topic_service');
       function($rootScope, gaTopic) {
         return {
           restrict: 'A',
-          templateUrl: function(element, attrs) {
-            return 'components/topic/partials/topic.' +
-              ((attrs.gaTopicUi == 'select') ? 'select.html' : 'html');
-          },
+          templateUrl: 'components/topic/partials/topic.html',
           scope: {},
           link: function(scope, element, attrs) {
+            var modalClsName = 'ga-topic-modal';
+            var modal = element.find('div.' + modalClsName);
```

Comment:
Use the `.modal` class instead`.   More generic and we don't need a constant to get the correct element.